osmnx/plot.py

```diff
@@ -664,7 +664,7 @@ def plot_footprints(  # noqa: PLR0913
 
 
 def plot_orientation(  # noqa: PLR0913
-    Gu: nx.MultiGraph,
+    Gu: nx.MultiGraph | nx.MultiDiGraph,
```

Comment:
@gboeing do we want to keep the argument name as `Gu` (assuming that u refers to undirected)? Applies here and other bearing functions?

Reply:
> assuming that u refers to undirected

Yes the convention we try to use consistently is that `Gu` means something like $G_u$: the undirected model of the network. Since this can now be either directed or undirected, you might as well rename the variable `G` for clarity.

Reply:
Also see prior comment on type annotation.